gulpfile.js

```diff
@@ -106,4 +107,15 @@ gulp.task('watch', function() {
     gulp.watch(files.cssMin, ['copy-css']);
     gulp.watch(files.js, ['minify-js']);
```

Comment:
Looks like we don't have the `minify-js` task anymore and we merged both in the `copy-js` task. If this is the case, we might want to replace the `minify-js` with `copy-js` and remove the next line

Reply:
@kaankabalak Thank you for for pointing out the issue. I will fix it shortly.